tests/resources/aria-at-test-run.mjs

```diff
@@ -0,0 +1,1252 @@
+              description: "for assertion",
+            },
+          ],
+          click: () =>
+            hooks.setCommandAssertion({commandIndex, assertionIndex, result: AssertionResultMap.FAIL_MISSING}),
+        }),
+      ],
+    });
+  }
+
+  /**
+   * @param {number} commandIndex
+   * @param {string} assertion
+   * @param {number} assertionIndex
+   */
+  function additionalAssertionResult(commandIndex, assertion, assertionIndex) {
+    const resultAdditionalAssertion = resultState.commands[commandIndex].additionalAssertions[assertionIndex];
+    return /** @type {InstructionDocumentResultsCommandsAssertion} */ ({
+      description: [
+        assertion,
+        {
+          required: true,
+          highlightRequired: resultAdditionalAssertion.highlightRequired,
+          description: "(required: mark support)",
+        },
+      ],
+      passChoice: assertionChoice(resultAdditionalAssertion, AdditionalAssertionResultMap.PASS, {
+        label: ["Good Support ", {offScreen: true, description: "for assertion"}],
+        click: () =>
+          hooks.setCommandAdditionalAssertion({
+            commandIndex,
+            additionalAssertionIndex: assertionIndex,
+            result: AdditionalAssertionResultMap.PASS,
+          }),
+      }),
+      failChoices: [
+        assertionChoice(resultAdditionalAssertion, AdditionalAssertionResultMap.FAIL_SUPPORT, {
+          label: ["No Support ", {offScreen: true, description: "for assertion"}],
+          click: () =>
+            hooks.setCommandAdditionalAssertion({
+              commandIndex,
+              additionalAssertionIndex: assertionIndex,
+              result: AdditionalAssertionResultMap.FAIL_SUPPORT,
+            }),
+        }),
+      ],
+    });
+  }
+}
+
+/**
+ * @typedef {typeof UserActionMap[keyof typeof UserActionMap]} UserAction
+ */
+
+export const UserActionMap = createEnumMap({
+  LOAD_PAGE: "loadPage",
+  OPEN_TEST_WINDOW: "openTestWindow",
+  CLOSE_TEST_WINDOW: "closeTestWindow",
+  VALIDATE_RESULTS: "validateResults",
+  CHANGE_TEXT: "changeText",
+  CHANGE_SELECTION: "changeSelection",
+  SHOW_RESULTS: "showResults",
+});
+
+/**
+ * @typedef {typeof UserObjectActionMap[keyof typeof UserObjectActionMap]} UserObjectAction
+ */
+
+export const UserObjectActionMap = createEnumMap({
+  FOCUS_UNDESIRABLE: "focusUndesirable",
+});
+
+/**
+ * @typedef {UserAction | UserActionFocusUnexpected} TestRunUserAction
+ */
+
+/**
+ * @typedef {EnumValues<typeof HasUnexpectedBehaviorMap>} HasUnexpectedBehavior
+ */
+
+export const HasUnexpectedBehaviorMap = createEnumMap({
+  NOT_SET: "notSet",
+  HAS_UNEXPECTED: "hasUnexpected",
+  DOES_NOT_HAVE_UNEXPECTED: "doesNotHaveUnexpected",
+});
+
+export const CommonResultMap = createEnumMap({
+  NOT_SET: "notSet",
+  PASS: "pass",
+});
+
+/**
+ * @typedef {EnumValues<typeof AdditionalAssertionResultMap>} AdditionalAssertionResult
+ */
+
+export const AdditionalAssertionResultMap = createEnumMap({
```

Comment:
nit: I certainly agree with the separation of concerns for the `Map`s here but it does feel like a few of these could be packaged into 1, especially the `...AssertionMap`s because they are related and it could limit the amount of documentation/'looking for property in wrong map' errors one could possibly encounter? Just a thought.

Reply:
Hmm. Yeah at this point I just wanted to mirror the values that get exported with ones in the internal state that follow a common idiom.